app/helpers/recruitment_cycle_helper.rb

```diff
@@ -0,0 +1,31 @@
+module RecruitmentCycleHelper
```

Comment:
Having a fair amount of logic in a helper is a bit smelly. One way to fix that would be to move a lot of these methods onto a `RecruitmentCycle` model (which is a plain Ruby object). Its initializer could take the cycle year and maybe Settings.current_cycle.

Reply:
We can leave `recruitment_cycle_title` here but move the rest into a model

Reply:
Will address in a subsequent PR.